src/components/MainSidebar/__tests__/__snapshots__/TimeDimensions.spec.js.snap

```diff
@@ -26,23 +26,11 @@ exports[`input type ENROLLMENT, WITH_REGISTRATION, custom labels 1`] = `
       <div
         class="label"
       >
-        le event date
-      </div>
-      <div
-        class="menuTrigger"
-      >
-        <svg
-          height="16"
-          viewBox="0 0 16 16"
-          width="16"
-          xmlns="http://www.w3.org/2000/svg"
+        <span
```

Comment:
The change to the snapshots is the removal of the `menuTrigger` button. Just confirming that is correct.